src/Document/Library/Bridge/Symfony/ValueResolver/UploadedFile.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Zenstruck\Document\Library\Bridge\Symfony\ValueResolver;
```

Comment:
```suggestion
namespace Zenstruck\Document\Library\Bridge\Symfony\Attribute;
```

Although, this doesn't have to be symfony-specific so maybe:

```suggestion
namespace Zenstruck\Document\Attribute;
```